src/ngraph_builder.cc

```diff
@@ -1142,23 +1173,14 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
             "Number of inputs is not 2 for ExpandDims");
       }
 
-      tf::Node* tf_input;
-      tf::Node* tf_dim;
-      TF_RETURN_IF_ERROR(op->input_node(0, &tf_input));
-      TF_RETURN_IF_ERROR(op->input_node(1, &tf_dim));
+      shared_ptr<ng::Node> ng_input;
+      TRY_FETCH_NG_OP(ng_input, 0);
 
-      auto ng_input = ng_op_map.find(tf_input->name());
-      if (ng_input == ng_op_map.end()) {
-        return tf::errors::InvalidArgument("Missing input: " +
-                                           tf_input->name());
-      }
-      auto ng_dim = ng_op_map.find(tf_dim->name());
-      if (ng_dim == ng_op_map.end()) {
-        return tf::errors::InvalidArgument("Missing input: " + tf_dim->name());
-      }
+      shared_ptr<ng::Node> ng_dim;
+      TRY_FETCH_NG_OP(ng_dim, 1);
```

Comment:
This type of mixed API adds to confusion. In some cases you are using GetNgOp and in some cases TRY_FETCH. Needs to be consistent.

Reply:
I added TRY_FETCH.. to remove some boilerplate code,  
` tf::Node* tf_input;
  TF_RETURN_IF_ERROR(op->input_node(0, &tf_input));
    auto ng_input = ng_op_map.find(tf_input->name());
     if (ng_input == ng_op_map.end()) {
        return tf::errors::InvalidArgument("Missing input: " +
                                           tf_input->name());
     }` is seen everywhere in the file. TRY_FETCH wraps GetNgOp to reduce few lines of code. Agree that it adds confusion, maybe let's address it later during refactoring

Reply:
Ok. Sounds good.